test/client/fixtures/sandbox/node/element-test.js

```diff
@@ -0,0 +1,14 @@
+test('Check the "scriptElementEvent" event is raised', function () {
+    window.script = document.createElement('script');
+
+    function handler (e) {
+        strictEqual(e.el, window.script);
+
+        hammerhead.off(hammerhead.EVENTS.scriptElementAdded, handler);
+    }
+
+    hammerhead.on(hammerhead.EVENTS.scriptElementAdded, handler);
+
+    eval(processScript('document.body.appendChild(window.script)'));
```

Comment:
We don't use code instrumentation for methods.
Just write `document.body.appendChild(script)`